pages/documentation/showcase.md

```diff
@@ -54,6 +54,7 @@ If your project is currently using USWDS and you would like to see it included i
 - [Digital Analytics Dashboard](https://analytics.usa.gov)
 - [Digital Service Playbook — U.S. Digital Service](https://playbook.cio.gov)
 - [Discovery Market Research Tool (GSA)](https://discovery.gsa.gov/)
+- [Export Compliance Assistance Program (ECAP)](https://nnsa.nsis.anl.gov/ecap/)
```

Comment:
**Question:** Should we order this to come _after_ EPA to keep it alphabetical? Or lead with ECAP?

Not sure what the best way to keep these ordered when it comes to acronyms!